### DISCUSSION TITLE
是否可以在插件页面增加翻译额度使用情况的显示

### DISCUSSION THREAD

**Discussion by DxPoly** [immersive-translate/immersive-translate#2249] [Ideas]
### 新功能 | New Feature

1.希望可以在最常用的插件页面，也就是点击插件后弹出的那个翻译功能弹窗里面，增加一个额度使用情况的显示，简单一点就足够了

### 为什么您想要添加这些功能 | Why you want to add these features

1.因为现在不能随时关注额度使用的情况，就会在一些可翻译可不翻译页面或者是 epub 文件的时候，使用翻译，然后导致额度用完。

**Comment by theowenyoung** [comment]
我们计划这个月内打通 DeepL 和 OpenAI 的翻译额度，使之共享 2000 万 tokens，届时额度问题应该能得到解决～

面板上的空间非常有限，应该不会添加此功能。
